### ISSUE TITLE
eulerAngles 

### ISSUE THREAD

**Issue by dariocavada** [olexale/arkit_flutter_plugin#70]
I've tried to rotate a cone, but **eulerAngles** seems not to work. The cone is in the same position with the same rotation.
Looking in the **NodeBuilder.swift** code, there is no **eulerAngles**. 
Also tried **rotation**, but it is not clear how it works ... maybe rotation with respect to the camera?
Following the code:

``` dart 
 _testRotation(ARKitImageAnchor anchor) {
    final anchorPos = anchor.transform.getColumn(3);
    final double rot = -pi / 2;
    final material =
        ARKitMaterial(diffuse: ARKitMaterialProperty(color: Colors.blue));

    final cone = ARKitCone(
        height: 0.1,
        topRadius: 0.00,
        bottomRadius: 0.02,
        materials: [material]);

    /* Node Cone */
    final nodeCone = ARKitNode(
      geometry: cone,
      position: vector.Vector3(anchorPos.x, anchorPos.y, anchorPos.z),
      eulerAngles: vector.Vector3(rot, rot, rot),
    );

    this.arkitController.add(nodeCone);



  }
```
Thanks in advance for the help.

![cone-no-rotation](https://user-images.githubusercontent.com/4304176/83901938-d378e700-a75b-11ea-94c5-af8e660514ef.jpeg)

**Comment by olexale** [comment]
That's an epic fail from my side. It seems that during huge refactoring of version 0.5 I've messed up with Euler angles.
It is possible to set them though as the method `onEulerAnglesChanged` is in place. Hence you may add a node to the scene and change its angles next line.

This issue will be autofixed as soon as I'll finish with the transformation matrix refactoring.
Sorry for the inconvenience.

**Comment by olexale** [comment]
This should be fixed in 0.6.0 as I, at last, had enough time to refactor the transformation matrix. Feel free to re-open the issue in case you face any problems. Thanks!
